Makefile.am

```diff
@@ -68,6 +68,7 @@ GENERATED_FILES = \
 	tests/metrics/network/network-metrics-cpu-consumption.sh \
 	tests/metrics/network/network-metrics-memory-pss.sh \
 	tests/metrics/network/network-metrics-memory-rss-1g.sh \
+	tests/metrics/network/network-metrics-memory-1g.sh \
```

Comment:
For consistency with the other two related tests, this really should be called `network-metrics-memory-pss-1g.sh` :-)
Could you fix that here and below (and obviously rename the source file itself) please.